src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/conda-store/config/conda_store_config.py

```diff
@@ -83,11 +88,301 @@ def conda_store_config(path="/var/lib/conda-store/config.json"):
 c.GenericOAuthAuthentication.user_data_key = "preferred_username"
 c.GenericOAuthAuthentication.tls_verify = False
 
+CONDA_STORE_ROLE_PERMISSIONS_ORDER = ["viewer", "developer", "admin"]
+
+
+@dataclasses.dataclass
+class CondaStoreNamespaceRole:
+    namespace: str
+    role: str
+
+
+@dataclasses.dataclass
+class KeycloakCondaStoreRoleScopes:
+    scopes: str
+    log: logging.Logger
+
+    def _validate_role(self, role):
+        valid = role in CONDA_STORE_ROLE_PERMISSIONS_ORDER
+        self.log.info(f"role: {role} is valid={valid}")
```

Comment:
`{role} is valid=False` seems a little strange.  Maybe `self.log.info(f"role: {role} is {'invalid' if valid else 'valid'}")`?